### ISSUE TITLE
Refactor the directory

### ISSUE THREAD

**Issue by version0chiro** [version0chiro/Ace-The-Code#2]
Arrange the directory for a better understanding

**Comment by ffricken** [comment]
I think it might be beneficial to add a descriptive file to each folder and maybe add test instances where applicable.

**Comment by version0chiro** [comment]
Yes, can help out a lot for other people. You can create an active issue for that, will try to work on it or someone else can also jump in.

**Comment by akankshatyagiat** [comment]
Hi @version0chiro, can I contribute to it?
I will try to refactor the leetcode directory.
